code/datums/elements/earhealing.dm

```diff
@@ -2,37 +2,33 @@
 	element_flags = ELEMENT_DETACH
 	var/list/user_by_item = list()
 
-/datum/element/earhealing/New()
-	START_PROCESSING(SSdcs, src)
-
 /datum/element/earhealing/Attach(datum/target)
 	. = ..()
 	if(!isitem(target))
 		return ELEMENT_INCOMPATIBLE
 
-	RegisterSignal(target, list(COMSIG_ITEM_EQUIPPED, COMSIG_ITEM_DROPPED), PROC_REF(equippedChanged))
+	RegisterSignal(target, list(COMSIG_ITEM_EQUIPPED, COMSIG_ITEM_DROPPED), PROC_REF(on_equip))
 
 /datum/element/earhealing/Detach(datum/target)
 	. = ..()
 	UnregisterSignal(target, list(COMSIG_ITEM_EQUIPPED, COMSIG_ITEM_DROPPED))
 	user_by_item -= target
 
-/datum/element/earhealing/proc/equippedChanged(datum/source, mob/living/carbon/user, slot)
+/datum/element/earhealing/proc/on_equip(datum/source, mob/living/carbon/user, slot)
 	SIGNAL_HANDLER
 
 	if(slot == ITEM_SLOT_EARS && istype(user))
+		START_PROCESSING(SSdcs, src)
 		user_by_item[source] = user
 	else
 		user_by_item -= source
 
 /datum/element/earhealing/process(delta_time)
 	for(var/i in user_by_item)
 		var/mob/living/carbon/user = user_by_item[i]
-		if(HAS_TRAIT(user, TRAIT_DEAF))
-			continue
 		var/obj/item/organ/ears/ears = user.getorganslot(ORGAN_SLOT_EARS)
-		if(!ears)
+		if(!ears || !ears.damage || ears.organ_flags & ORGAN_FAILING)
```

Comment:
Bit operations need to be wrapped in parameters. If they aren't it can quickly become confusing as to what they are operating on.